assessment_software/src/app/navbar/navbar.component.css

```diff
@@ -0,0 +1,15 @@
+.navbar-container{
+    @apply  bg-white border-gray-200 border border-solid rounded px-2 text-sm font-sans fixed top-1 left-1 right-1 grid grid-cols-4 gap-2 items-center;
```

Comment:
add z-index to the navbar component since we want it to be at the top of everything. -z-50